modin/pandas/series.py

```diff
@@ -1343,7 +1355,7 @@ def shift(
         """
         Shift index by desired number of periods with an optional time `freq`.
         """
-        # pandas 2.1.0rc0 ignores suffix parameter
+        # pandas 2.1.0rc0 ignores suffix parameter (https://github.com/pandas-dev/pandas/issues/54806)
```

Comment:
```suggestion
        # pandas 2.1.0 ignores suffix parameter (https://github.com/pandas-dev/pandas/issues/54806)
```
(as we no longer install RC in the `.yml` files)